packages/app/cypress/e2e/runner/retries.mochaEvents.snapshots.ts

```diff
@@ -4880,527 +5074,7 @@ export const snapshots = {
         'id': 'r1',
         'title': '',
         'root': true,
-        'type': 'suite',
-        'file': 'relative/path/to/spec.js',
-        'retries': -1,
-        '_slow': 10000,
-      },
-    ],
-    [
-      'mocha',
-      'end',
-      {
-        'end': 'match.date',
-      },
-    ],
-  ],
-
-  'src/cypress/runner retries mochaEvents cant retry from [after] #1': [
```

Comment:
stale snapshots -- no corresponding test using it. Tests were not ported over in v10 uplift in https://github.com/cypress-io/cypress/pull/20678 and then deleted in https://github.com/cypress-io/cypress/pull/21638